packages/editor/src/components/global-keyboard-shortcuts/visual-editor-shortcuts.js

```diff
@@ -169,35 +160,32 @@ export default compose( [
 			selectedBlockClientIds,
 		};
 	} ),
-	withDispatch( ( dispatch, ownProps, { select } ) => {
+	withDispatch( ( dispatch ) => {
 		const {
 			clearSelectedBlock,
 			multiSelect,
 			redo,
 			undo,
 			removeBlocks,
-			savePost,
 		} = dispatch( 'core/editor' );
 
 		return {
-			onSave() {
-				// TODO: This should be handled in the `savePost` effect in
-				// considering `isSaveable`. See note on `isEditedPostSaveable`
-				// selector about dirtiness and meta-boxes.
-				//
-				// See: `isEditedPostSaveable`
-				const { isEditedPostDirty } = select( 'core/editor' );
-				if ( ! isEditedPostDirty() ) {
-					return;
-				}
-
-				savePost();
-			},
 			clearSelectedBlock,
 			onMultiSelect: multiSelect,
 			onRedo: redo,
 			onUndo: undo,
 			onRemove: removeBlocks,
 		};
 	} ),
-] )( EditorGlobalKeyboardShortcuts );
+] )( VisualEditorGlobalKeyboardShortcuts );
+
+export default EnhancedVisualEditorGlobalKeyboardShortcuts;
+
+export function EditorGlobalKeyboardShortcuts() {
+	deprecated( 'EditorGlobalKeyboardShortcuts', {
+		alternative: 'VisualEditorGlobalKeyboardShortcuts',
+		plugin: 'Gutenberg',
```

Comment:
I wonder if we should still be using `plugin` now that Gutenberg and WP Core are essentially the same codebase?

(We do this elsewhere, so just a question 🙂)